### PR TITLE
fix height of sponsors

### PR DESCRIPTION
**Pull request by gurgunday**
fixes:

Screenshot 2024-07-09 at 14 48 20

I'm not sure if a CSS library is used here, so I directly added `height: 100%`, but I can also use classes if that's better

### CODE REVIEW

**Comment by gurgunday** [comment]
@Eomm can I get a deployment? 😺

File: src/pages/index.jsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default function Home() {
           <div className="container">
             <div className="row">
               <div className="col col--6">
-                <div className="alternate-sponsors">
+                <div className="alternate-sponsors" style={{ display: '100%' }}>
```
Comment (climba03003) [inline src/pages/index.jsx]: Is it a valid CSS?
Shouldn't `display` accept `block`, `inline-block`, etc.
I never see `display: 100%`

Better add it under https://github.com/fastify/website/blob/fc8f5149416b207181f32417de0d52bdeca0087e/src/css/custom.css#L70-L81

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: src/pages/index.jsx (hunk 1 above)
Comment (gurgunday) [reply src/pages/index.jsx]: I don’t know why it became display lol, it should be height 100%

**Review by climba03003 — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**